allure-report-face/src/variables.css

```diff
@@ -1,10 +1,10 @@
 $brand-blue: #0044bb;
 $brand-red: #ff0000;
 
-$color-passed: #3bbd25;
-$color-failed: #ff0000;
-$color-broken: #ffa924;
-$color-canceled: #999999;
+$color-passed: #97cc64;
+$color-failed: #fd5a3e;
+$color-broken: #ffd963;
```

Comment:
can see anything on this yellow
![screenshot from 2016-05-20 11-34-03](https://cloud.githubusercontent.com/assets/812240/15423657/ce3026ec-1e7e-11e6-814f-41e547a2e18d.png)